### ISSUE TITLE
Enable asset inventory processor customized scope in Mapping Server

### ISSUE THREAD

**Issue by capri-xiyue** [abcxyz/pmap#137]
### TL;DR

Currently, https://github.com/abcxyz/pmap/blob/5b24e38b19e0c8b8e12854ead228111b16e503af/pkg/cli/mapping_server.go#L123, mapping server defaults to use the project as default scope of Asset Inventory Processor. Customers may want to use folder id or organization id as the default scope of Asset Inventory Processor（otherwise customers may not be able register resources like gcs bucket whose resource name doesn't include project). We need to enable asset inventory processor customized scope in Mapping Server. Similar to how JVS users can customize the grace period, disable period and so on.

**Comment by capri-xiyue** [comment]
Not sure whether it is a common case that users will have resources under multiple orgs. For example, users have gcs buckets under orgA and orgB. But for this issue, let's just make it work for only one org. We can change it to support multiple orgs if needed

**Comment by sailorlqh** [comment]
@capri-xiyue I think we can't search resources under other orgs, according to [searchAllResouces](https://pkg.go.dev/cloud.google.com/go/asset/apiv1#Client.SearchAllResources), the API only allow up to search resource within scope of `Project`, `Folder`, `organization`. So each org should have it's own pmap service to find resources.

Please correct me if I am wrong.

**Comment by capri-xiyue** [comment]
> > Not sure whether it is a common case that users will have resources under multiple orgs. For example, users have gcs buckets under orgA and orgB. But for this issue, let's just make it work for only one org. We can change it to support multiple orgs if needed
> 
> @capri-xiyue I think we can't search resources under other orgs, according to [searchAllResouces](https://pkg.go.dev/cloud.google.com/go/asset/apiv1#Client.SearchAllResources), the API only allow us to search resource within scope of `Project`, `Folder`, `organization`. So each org should have it's own pmap service to find resources.
> 
> Please correct me if I am wrong.

The input could be a list orgs/folders and we call CAIS api multiple times(one time for each org/folder)
